package.json

```diff
@@ -48,7 +48,8 @@
     ]
   },
   "dependencies": {
-    "clsx": "^1.1.1"
+    "clsx": "^1.1.1",
+    "postcss-preset-env": "^9.3.0"
```

Comment:
I think it should be in devDependencies

Reply:
I'll move that to dev dependency's and change the behavior to use a flag!